Testnet/StandardToken/StandardToken.Tests/StandardTokenTests.cs

```diff
@@ -18,7 +17,7 @@ public class StandardTokenTests
         private Address destination;
         private string name;
         private string symbol;
-        private uint decimals;
+        private byte decimals;
```

Comment:
We tests for get/set decimals. From my local tests in other contracts, this implementation doesn't work as expected.